manifests/agent.pp

```diff
@@ -32,6 +39,10 @@
     content => "${percent_changed_facts}",
   }
 
+  service { 'crond':
```

Comment:
@doug-rosser also not sure if this is a good idea (although just realized I need to use the parameter I created for it.